source/_integrations/matrix.markdown

```diff
@@ -158,6 +161,8 @@ notify:
   - name: NOTIFIER_NAME
     platform: matrix
     default_room: ROOM_ID_OR_ALIAS
+    default_markdown: true|false
+    default_notice: true|false
```

Comment:
These are not valid values, should be either `true` or `false`.

Reply:
Changed to false